packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -141,18 +141,24 @@ export default function OrgNav({
         />
 
         <ul className={ulContainer}>
-          {paymentVariant && current?.paymentStatus && (
-            <li>
-              <Badge
-                className={css`
-                  margin-right: 25px;
-                `}
-                variant={paymentVariant}
-              >
-                {current.paymentStatus.toUpperCase()}
-              </Badge>
-            </li>
-          )}
+          {current &&
+            current.paymentStatus &&
```

Comment:
Could you simplify this to `current?.paymentStatus ...etc`?

Reply:
Trying to move away from chaining for @DesignerDave's rollup PR